charts/flyte-core/values-onprem.yaml

```diff
@@ -0,0 +1,841 @@
+#      enabled: true
+#      auth:
+#        type: "file_path"
+#        tokenPath: "/path/to/testcluster3/token"
+#        certPath: "/path/to/testcluster3/cert"
+
+  # -- Configuration for Flyte console UI
+  console:
+    BASE_URL: /console
+    CONFIG_DIR: /etc/flyte/config
+  
+  logger:
+    show-source: true
+    level: 6
+  
+  # -- Domains configuration for Flyte projects. This enables the specified number of domains across all projects in Flyte.
+  domain:
+    domains:
+      - id: development
+        name: development
+      - id: staging
+        name: staging
+      - id: production
+        name: production
+
+  # Refer to the full [structure](https://pkg.go.dev/github.com/lyft/flyteadmin@v0.3.37/pkg/runtime/interfaces#ApplicationConfig) for documentation.
+  schedulerConfig:
+    scheduler:
+      metricsScope: "flyte:"
+      profilerPort: 10254
+
+  # -- FlyteAdmin server configuration
+  adminServer:
+    # Refer to the [server config](https://pkg.go.dev/github.com/lyft/flyteadmin@v0.3.37/pkg/config#ServerConfig).
+    server:
+      httpPort: 8088
+      grpcPort: 8089
+      security:
+        # -- Controls whether to serve requests over SSL/TLS.
+        secure: false
+        # -- Controls whether to enforce authentication. Follow the guide in https://docs.flyte.org/ on how to setup authentication.
+        useAuth: true
+        allowCors: true
+        allowedOrigins:
+          # Accepting all domains for Sandbox installation
+          - "*"
+        allowedHeaders:
+          - "Content-Type"
+          - "flyte-authorization"
+    # Refer to the full [structure](https://pkg.go.dev/github.com/lyft/flyteadmin@v0.3.37/pkg/runtime/interfaces#ApplicationConfig) for documentation.
+    flyteadmin:
+      roleNameKey: "iam.amazonaws.com/role"
+      profilerPort: 10254
+      metricsScope: "flyte:"
+      metadataStoragePrefix:
+        - "metadata"
+        - "admin"
+      eventVersion: 2
+      testing:
+        host: http://flyteadmin
+    # -- Authentication configuration
+    auth:
+      grpcAuthorizationHeader: "flyte-authorization"
+      authorizedUris:
+        # This should point at your public http Uri.
+        - https://<flyte ingress url>
+        # This will be used by internal services in the same namespace as flyteadmin
+        - http://flyteadmin:80
+        # This will be used by internal services in the same cluster but different namespaces
+        - http://flyteadmin.flyte.svc.cluster.local:80 #assuming that flyte is installed in flyte namespace
+
+      # Controls app authentication config
+      appAuth:
+
+        authServerType: External
+        externalAuthServer:
+          allowedAudience: flytepropeller
+          baseUrl: "" # populate this value only if different from userauth baseUrl
+          metadataUrl: ".well-known/openid-configuration"
+
+
+        thirdPartyConfig:
+          flyteClient:
+            audience: flytepropeller
+            clientId: flytepropeller
```

Comment:
This is only required in case flyteClients are being run from a machine which doesn't have browser access. 
We can mention this info here and also have an alternate config in comments which shows with browser access.

Reply:
yes